### PR TITLE
Attendance marking page

### LINKED ISSUE #11: [Impl] Implement attendance marking page for students.
Create the student attendance marking page.

You can use the fingerprint component https://github.com/cse-mosip/Exam-Attendance-UI/issues/5 and the custom sign in component for the student https://github.com/cse-mosip/Exam-Attendance-UI/issues/8

Refer the **attendance collecting page**  and **student sign in via index page** in [Figma design](https://www.figma.com/file/ewqkiF5GG353Ae8aP2f3df/Scheduled-Page?type=design&node-id=0%3A1&mode=design&t=iQUlSYpAO1E7ueXP-1)

### PR DESCRIPTION
**Pull request by niruthikka24**

### CODE REVIEW

**Comment by Malith-19** [comment]
There are some conflicts guys. Merge the main into this branch or resolve the conflicts here using the web editor.

File: src/pages/attendance-marking-page.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { Button, TextField, Link } from "@mui/material";
+import * as React from "react";
```
Comment (Malith-19) [inline src/pages/attendance-marking-page.js]: We don't need to import all from the react right? I think we can remove this import. Check the other places too.

Comment (gnishaanthini) [reply src/pages/attendance-marking-page.js]: removed the unnecessary imports

**Review by Malith-19 — APPROVED** [review]

File: src/components/fingerprint-scanner.js [hunk 2]
```diff
@@ -40,20 +41,42 @@ const FingerprintScanner = ({ isStudent, setIsBioLogin }) => {
...
+//   letterSpacing: "1px",
+//   fontFamily: "Alata, sans-serif",
+//   fontWeight: 400,
+//   border: 1,
+//   borderRadius: 5,
+//   width: "35%",
+//   padding: "1%",
+//   boxShadow: "0px 4px 10px rgba(0, 0, 0, 0.1)",
+// };
+
```
Comment (chathuranga-jayanath-99) [inline src/components/fingerprint-scanner.js]: we can remove these commented lines

File: src/pages/attendance-marking-page.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+export default function AttendanceMarkingPage({isStudent}) {
+
+  const [isBioLogin, setIsBioLogin] = useState(true);
+
+  const handleFlip = () => {
+    setIsBioLogin(!isBioLogin)
+  }
+
+
```
Comment (chathuranga-jayanath-99) [inline src/pages/attendance-marking-page.js:14]: we can remove additional new lines. check other places as well.

File: src/pages/custom-login-page.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { Button, TextField, Link } from "@mui/material";
 import * as React from "react";
```
Comment (chathuranga-jayanath-99) [inline src/pages/custom-login-page.js]: can't we remove this line?

Comment (niruthikka24) [reply src/pages/custom-login-page.js]: Unnecessary imports, new lines and comments removed.

**Review by Malith-19 — APPROVED** [review]

**Review by chathuranga-jayanath-99 — APPROVED** [review]

**Pull request merged**